apps/vr-tests/.storybook/preview.js

```diff
@@ -35,3 +42,19 @@ setAddon({
     return this;
   },
 });
+
+/**
+ * @type {import('@storybook/react').Meta['decorators']}
+ */
+export const decorators = [removeCanvasInlineStyles];
+
+/**
+ * Temporary solution to remove inline styles injected by new default SB layout (https://storybook.js.org/docs/react/configure/story-layout)
+ * TODO - remove this once we migrate to SB 6.1
```

Comment:
It looks like [SB 6.1](https://github.com/storybookjs/storybook/issues/11884) is already released, are we going to do a follow up PR to migrate to 6.1? Or should we just move to 6.1 right now?

Reply:
From my experience  doing several bumps to major releases tends to introduce even more issues. thus I prefer step by step (this PR). Unfortunately because bad SB docs there was more digging needed which is now resolved and documented. We can migrate to 6.1 after this lands in master (@bsunderhus  ran into other issues when he tried to migrate directly to 6.1)

Reply:
Sounds good 👍